src/main/java/connectfour/EndController.java

```diff
@@ -0,0 +1,77 @@
+package connectfour;
+
+import javafx.application.Platform;
+import javafx.fxml.FXML;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.control.Button;
+import javafx.scene.layout.GridPane;
+import javafx.scene.paint.Color;
+import javafx.scene.text.Text;
+import javafx.stage.Stage;
+
+import java.io.IOException;
+
+public class EndController {
+
+    @FXML
+    private Button playAgainButton;
+
+    @FXML
+    private Button exitButton;
+
+    @FXML
+    private Text winnerNameText;
+
+    @FXML
+    private Text pointsInGame;
+
+    @FXML
+    private GridPane textContainer;
```

Comment:
Our naming convention requires that every name of Object JavaFX should be started by its class. For example: 
`private Text textWinnerName;`